javascript/organizing_your_javascript_code/es6_modules.md

```diff
@@ -64,7 +64,7 @@ There are a couple of key concepts to understanding how webpack works - __entry_
 
 Now that we (sorta) understand what webpack is doing it's time to discuss the module syntax. There are only 2 components to it - __import__ and __export__.
 
-- Take a moment to look at the docs for [import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and [export](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export).
+- Take a moment to look at the [MDN documentation about the import deceleration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and the [MDN documentation about the export deceleration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) which give great examples for the different syntax on import and export.
```

Comment:
Would it be possible to frame this in a "softer" manner? Learners may consider this to be a "must-read" and get demotivated/ be unsure about having to read the full documentation on imports/exports without a simpler introduction, first.

Reply:
In general, the issue with the specific documentation at the beginning of the lesson's section on these statements may be a deterrent, which's why they were *specifically* mentioned at the end. While the previous arrangement may not be perfect - can you think of anything better?

Reply:
Actually they were linked in both places, but I have some ideas to help.  I will work on it when back at computer

Reply:
I added a new commit, what do you think?

Reply:
LGTM! I like it. Feel free to close this, I love the new version. No stray links, and very to-the-point.